NEWS.md

```diff
@@ -1,5 +1,7 @@
 # pkgdown (development version)
 
+* pkgdown, for Bootstrap 4, supports tabsets in articles [as in R Markdown](https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html) including [fading effect](https://bookdown.org/yihui/rmarkdown/html-document.html#tabbed-sections) (@JamesHWade, #1667).
```

Comment:
I'm quite sure the HTML structure is now ok for fading, but there's some JS missing, somehow.

Reply:
https://github.com/rstudio/bslib/issues/324

Reply:
Fixed the markup I had added :woman_facepalming: https://getbootstrap.com/docs/4.6/components/navs/#javascript-behavior